OstelcoStyles/OstelcoLabel.swift

```diff
@@ -123,6 +123,16 @@ public class DataAmountOnHomeLabel: OstelcoLabel {
     }
 }
 
+public class DataRemainingLabel: OstelcoLabel {
```

Comment:
Where is this used?  we already have `DataAmountOnHomeLabel`  to show the balance

Reply:
It's the little "Left" label underneath that.